### PR TITLE
PFW-1386 Address TODO in M600 Z-axis recovery

### PR DESCRIPTION
**Pull request by gudnimg**
Resolves a TODO left from last week's https://github.com/DRracer/Prusa-Firmware/pull/66/files#diff-a2e2d6e9dab1e37ea9a2882433e2b8f70af751b3c3445d5e997d11a93d4b28b3

### CODE REVIEW

**Review by DRracer — APPROVED** [review]

**Comment by DRracer** [comment]
@gudnimg nice :+1: . Do we wait for merge of #70 before merging this PR?

**Comment by gudnimg** [comment]
@DRracer This PR should be fine to merge separately. I don't see any conflicts.

**Pull request merged**
